examples/pytorch/question-answering/run_qa.py

```diff
@@ -14,7 +14,89 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """
-Fine-tuning the library models for question answering.
+Example script for integrating spaseml with the transformers library.
```

Comment:
I'm torn on this, but I'd rather not update the original documentation for the script in the event that we can push this up to the original repository